code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -502,7 +504,7 @@ SUBSYSTEM_DEF(dynamic)
 // Called BEFORE everyone is equipped with their job
 /datum/controller/subsystem/dynamic/proc/pre_setup()
 	if(CONFIG_GET(flag/dynamic_config_enabled))
-		var/json_file = file("[global.config.directory]/dynamic.json")
+		var/json_file = file("[global.config.directory]/[GLOB.dynamic_config_string]")
```

Comment:
There's a ton of things we configure in dynamic.json that aren't rulesets. Really don't think we should be overriding *the entire thing*.

I think better way to do this is a ruleset flag/similar that says "I don't run on station wide background checks"

Reply:
Yeah I'll start implementing this. I may do it inclusively though, makes it easier to implement unique rulesets